app/client/cli/account.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/utility/types"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(NewAccountCommand())
+}
+
+func NewAccountCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Account",
+		Short:   "Account specific commands",
+		Aliases: []string{"account"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(accountCommands()...)
+
+	return cmd
+}
+
+func accountCommands() (cmds []*cobra.Command) {
+
+	sendCmd := &cobra.Command{
+		Use:     "Send <from> <to> <amount>",
+		Short:   "Send <from> <to> <amount>",
+		Long:    "Sends <amount> to address <to> from address <from>",
+		Aliases: []string{"send"},
+		Args:    cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(deblasis): update when we have keybase
```

Comment:
For keybase related followups, you can even do `TODO(pocket/issues/150)`

Reply:
done